<?xml version="1.0"?>
<launch>
	<include file="$(find keti_robot_moveit_config)/launch/demo_gazebo.launch"/>
    <node pkg="keti_robot_control" type="robot_control" name="robot_control_node" output="screen" />

	<include file="$(find keti_robot_server)/launch/robot_server.launch"/>

	<include file="$(find keti_robot_control_gui)/launch/robot_control_gui.launch"/>
</launch>
